tests/test_helpers.py

```diff
@@ -186,3 +186,49 @@ def test_array_association():
 
     with pytest.raises(ValueError, match='not supported.'):
         mesh.get_array_association('name', preference='row')
+
+
+def test_set_default_active_vectors():
+    mesh = pyvista.Sphere()
+    mesh.clear_data()  # make sure we have a clean mesh with no arrays to start
+
+    assert mesh.active_vectors_name is None
+
+    # Point data vectors
+    mesh["vec_point"] = np.ones((mesh.n_points, 3))
+    pyvista.set_default_active_vectors(mesh)
+    assert mesh.active_vectors_name == "vec_point"
+    mesh.clear_data()
+
+    # Cell data vectors
+    mesh["vec_cell"] = np.ones((mesh.n_cells, 3))
+    pyvista.set_default_active_vectors(mesh)
+    assert mesh.active_vectors_name == "vec_cell"
+    mesh.clear_data()
+
+    # Raises if no data is present
+    with pytest.raises(ValueError):
+        pyvista.set_default_active_vectors(mesh)
+    assert mesh.active_vectors_name is None
+
+    # Raises if no vector-like data is present
+    mesh["scalar_data"] = np.ones((mesh.n_points, 1))
+    with pytest.raises(ValueError):
+        pyvista.set_default_active_vectors(mesh)
+    assert mesh.active_vectors_name is None
+    mesh.clear_data()
+
+    # Raises if multiple vector-like data is present
+    mesh["vec_data1"] = np.ones((mesh.n_points, 3))
+    mesh["vec_data2"] = np.ones((mesh.n_points, 3))
+    with pytest.raises(ValueError):
+        pyvista.set_default_active_vectors(mesh)
+    assert mesh.active_vectors_name is None
+    mesh.clear_data()
+
+    # Raises if multiple vector-like data in cell and point
+    mesh["vec_data1"] = np.ones((mesh.n_points, 3))
+    mesh["vec_data2"] = np.ones((mesh.n_cells, 3))
```

Comment:
I would use the same array name for both, just for some edge case paranoia.
```suggestion
    mesh["vec_data"] = np.ones((mesh.n_points, 3))
    mesh["vec_data"] = np.ones((mesh.n_cells, 3))
```
Not that I see anything in the implementation that would fail if the names are the same.

Reply:
I added a new section to the test instead of replacing.